docs/Reference/Reply.md

```diff
@@ -199,6 +202,47 @@ reply.getHeader('x-foo') // undefined
 
 Returns a boolean indicating if the specified header has been set.
 
+### .trailer(key, function)
+<a id="trailer"></a>
+
+Sets a response trailer.
```

Comment:
Please
document in which moments we call the functions as well as that the functions must be synchronous.